recipes-devtools/rust/rust-snapshot-1.24.1.inc

```diff
@@ -4,7 +4,7 @@
 ## snapshot info is taken from rust/src/stage0.txt
 ## TODO: find a way to add additional SRC_URIs based on the contents of an
 ##       earlier SRC_URI.
-RS_VERSION = "1.23.0"
+RS_VERSION = "${PV}"
```

Comment:
This will break all other targets.

Reply:
Oh I see. You change this and this causes you to need the `fix-crate-ver-check.patch`. This isn't correct.